avm/res/compute/virtual-machine-scale-set/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.27.1.19265",
-      "templateHash": "7041078491413360342"
+      "version": "0.26.54.24096",
```

Comment:
Hey @rahalan, please not you downgraded the bicep version and the latest is even `0.28.1`. I'd recommend a qick update and re-running the `Set-AVMModule` scipt as I 'think' it also does something with the formatting.

Reply:
fixed